### PR TITLE
Update Style- and ThemeManager descriptions

### PR DESCRIPTION
**Pull request by doishub**

### CODE REVIEW

File: meta/contao-thememanager/core/en.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+en:
+    title: Contao ThemeManager Core
+    description: >
+        The Contao ThemeManager provides a basic framework for building any kind of theme. Several tools are included
+        that allow you to compile SCSS / CSS files or edit their SCSS variables directly from the back end. The framework
+        can be completely complemented by other extensions. The core extension already includes some basic styles such
+        as coloring, text properties or a grid system, which can easily be replaced by another can be.
```
Comment (xchs) [inline meta/contao-thememanager/core/en.yml:7]:
```suggestion
        as coloring, text properties or a grid system, which can easily be replaced by another one.
```

File: meta/contao-thememanager/core/de.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+de:
+    title: Contao ThemeManager Core
+    description: >
+        Der Contao ThemeManager liefert ein Grundgerüst für den Aufbau beliebiger Themes. Dabei werden verschiedene
+        Werkzeuge mitgeliefert, welche bspw. für die Kompilierung von SCSS / CSS Dateien oder die Bearbeitung dessen
```
Comment (xchs) [inline meta/contao-thememanager/core/de.yml:5]:
```suggestion
        Werkzeuge mitgeliefert, welche bspw. für die Kompilierung von SCSS-/CSS-Dateien oder die Bearbeitung dessen
```

File: meta/contao-thememanager/core/en.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+en:
+    title: Contao ThemeManager Core
+    description: >
+        The Contao ThemeManager provides a basic framework for building any kind of theme. Several tools are included
+        that allow you to compile SCSS / CSS files or edit their SCSS variables directly from the back end. The framework
```
Comment (xchs) [inline meta/contao-thememanager/core/en.yml:5]:
```suggestion
        that allow you to compile SCSS/CSS files or edit their SCSS variables directly from the back end. The framework
```

File: linter/whitelists/de.txt [hunk 4]
```diff
@@ -30,7 +30,6 @@ ausserdem
 Automatisierungsaufgaben
 Avatar
 Avatare
-Backend
```
[the remark this answers is not in the document]
Comment (doishub) [reply linter/whitelists/de.txt:33]: I thought the spelling had to be "back end" - I will take it up again.

**Comment by doishub** [comment]
Thank you, I have adapted everything as requested.

File: linter/whitelists/de.txt (hunk 4 above)
[the remark this answers is not in the document]
Comment (xchs) [reply linter/whitelists/de.txt]: It is "**Backend**" in German and "**back end**" in English.

**Comment by xchs** [comment]
Thank you @doishub

**Pull request merged**